front/lib/api/assistant/actions/dust_app_run.ts

```diff
@@ -357,8 +361,12 @@ export async function* runDustApp(
   // Let's run the app now.
   const now = Date.now();
 
-  const prodCredentials = await prodAPICredentialsForOwner(owner);
-  const api = new DustAPI(prodCredentials);
+  const prodCredentials = await prodAPICredentialsForOwner(owner, {
```

Comment:
I feel this is becoming confusing. Why is it "prod api credentials" then ? if we're already in prod, then it's just "regular" credentials, and if we're not in prod and it's using local, then it's not prod credentials ?

Reply:
This is always prod unless that new special flag